src/main/java/com/spartronics4915/frc2019/subsystems/Superstructure.java

```diff
@@ -214,7 +214,10 @@ public void onLoop(double timestamp)
                     /* Placing/intaking game pieces */
                     case INTAKING_AND_ALIGNING_CLOSEST_FORWARD_TARGET:
                         mCargoIntake.setWantedState(CargoIntake.WantedState.INTAKE);
-                        mCargoChute.setWantedState(CargoChute.WantedState.BRING_BALL_TO_TOP);
+                        if (mCargoChute.ballInPosition())
```

Comment:
I think we should add the `BRING_BALL_TO_TOP` back instead of moving this to `Superstructure` (see my above comment about this). The driver needs this automation too.